runtime/vm/createramclass.cpp

```diff
@@ -2921,11 +2921,25 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 	J9Module* module = NULL;
 #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
 	UDATA romFieldCount = romClass->romFieldCount;
-	UDATA flattenedClassCacheAllocSize = sizeof(J9FlattenedClassCache) * (romFieldCount + 1);
+	/*             Structure of Flattened Class Cache              */
+	/***************************************************************
```

Comment:
I dont think this picture is needed anymore since the structure is more clear now